meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/apalis-imx8/lmp-base.cfg

```diff
@@ -3,8 +3,8 @@ CONFIG_SPL_OF_CONTROL=y
 CONFIG_OF_LIBFDT_OVERLAY=y
 CONFIG_FIT=y
 CONFIG_FIT_SIGNATURE=y
-CONFIG_FIT_VERBOSE=y
 CONFIG_SPL_FIT_SIGNATURE=y
+CONFIG_FIT_VERBOSE=y
```

Comment:
order change not needed

Reply:
This was to make the comparison between files match up better.

See commit title: bsp: u-boot-fio: apalis-imx8: sync updates from imx8qm-mek